.github/workflows/ci.yml

```diff
@@ -0,0 +1,136 @@
+name: CI
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  # Cabal
+  cabal:
+    name: Cabal / GHC ${{ matrix.ghc }} ${{ matrix.project-variant }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # TODO: "9.2.8", "9.4.7", "9.6.3"
+        ghc: ["8.0.2", "8.2.2", "8.4.4", "8.6.5", "8.8.4", "8.10.7", "9.0.2"]
+        project-variant: [""]
+        include:
+          - ghc: 8.6.5
+            project-variant: -lower
```

Comment:
My suggestion is to do this for GHC 8.0.2 instead, test the oldest of the oldest.

Reply:
I _just_ noticed, one sec